lib/ProductOpener/Display.pm

```diff
@@ -917,7 +940,7 @@ sub analyze_request($request_ref) {
 	elsif ((defined $options{redirect_texts}) and (defined $options{redirect_texts}{$lang . "/" . $components[0]})) {
 		$request_ref->{redirect} = $formatted_subdomain . "/" . $options{redirect_texts}{$lang . "/" . $components[0]};
 		$log->info("renamed text, redirecting", { textid => $components[0], redirect => $request_ref->{redirect} }) if $log->is_info();
-		return 301;
+		redirect(302, $request_ref->{redirect});
```

Comment:
the change from 301 to 302 is intentional, right ?

Reply:
yes, 301s make things harder to test and debug and to fix mistakes